src/printer.js

```diff
@@ -2309,7 +2294,7 @@ function genericPrintNoParens(path, options, print, args) {
     case "TSDeclareKeyword":
       return "declare"
     case "TSModuleBlock":
-      return concat(path.map(print, "body"))
+      return join(hardline, path.map(print, "body"));
```

Comment:
You probably want `printStatementSequence` instead. It handles preserving empty lines and handling `;`